packages/client/src/helpers/client/interceptors/authentication/AxiosAuthenticationTokenManager.ts

```diff
@@ -388,7 +415,7 @@ class AxiosAuthenticationTokenManager {
    * @param useCache - Returns an access token from the cache if available, if not a new create access token request will be made.
    * @returns Promise that will resolve with a renewed or cached access token from the current provider. If the renew request fails, the promise will reject with the error.
    */
-  async getAccessToken(useCache) {
+  async getAccessToken(useCache: boolean) {
```

Comment:
You can add a default value of `true` if useCache parameter is not passed as it is the way both the GuestTokenProvider and UserTokenProvider methods are defined.

Reply:
done.